steps/cadence-innovus-power-includes-sky130/outputs/power-strategy-singlemesh.tcl

```diff
@@ -0,0 +1,122 @@
+#=========================================================================
+# power-strategy-singlemesh.tcl
+#=========================================================================
+# This script implements a single power mesh on the upper metal layers.
+# Note that M2 is expected to be vertical, and the lower metal layer of
+# the power mesh is expected to be horizontal.
+#
+# Author : Christopher Torng
+# Date   : January 20, 2019
+
+#-------------------------------------------------------------------------
+# Stdcell power rail preroute
+#-------------------------------------------------------------------------
+# Generate horizontal stdcell preroutes
+
+sroute -nets {VDD VSS}
+
+#-------------------------------------------------------------------------
+# Shorter names from the ADK
+#-------------------------------------------------------------------------
+
+set pmesh_bot $ADK_POWER_MESH_BOT_LAYER
+set pmesh_top $ADK_POWER_MESH_TOP_LAYER
+
+#-------------------------------------------------------------------------
+# Power ring
+#-------------------------------------------------------------------------
+
+addRing -nets {VDD VSS} -type core_rings -follow core   \
+        -layer [list top  $pmesh_top bottom $pmesh_top  \
+                     left $pmesh_bot right  $pmesh_bot] \
+        -width $savedvars(p_ring_width)                 \
+        -spacing $savedvars(p_ring_spacing)             \
+        -offset $savedvars(p_ring_spacing)              \
+        -extend_corner {tl tr bl br lt lb rt rb}
+
+#-------------------------------------------------------------------------
+# Power mesh bottom settings (vertical)
+#-------------------------------------------------------------------------
+# - pmesh_bot_str_width            : 8X thickness compared to 3 * M1 width
+# - pmesh_bot_str_pitch            : Arbitrarily choosing the stripe pitch
+# - pmesh_bot_str_intraset_spacing : Space between VSS/VDD, choosing
+#                                    constant pitch across VSS/VDD stripes
+# - pmesh_bot_str_interset_pitch   : Pitch between same-signal stripes
+
+# Get M1 min width and signal routing pitch as defined in the LEF
+
+set M1_min_width    [dbGet [dbGetLayerByZ 2].minWidth]
+set M1_route_pitchX [dbGet [dbGetLayerByZ 2].pitchX]
+
+# Bottom stripe params
+
+set pmesh_bot_str_width [expr  8 *  3 * $M1_min_width   ]
+set pmesh_bot_str_pitch [expr 4 * 10 * $M1_route_pitchX]
```

Comment:
This used to be 10, but it was arbitrary:

```
54c54
< set pmesh_bot_str_pitch [expr 10 * 10 * $M1_route_pitchX]
---
> set pmesh_bot_str_pitch [expr 4 * 10 * $M1_route_pitchX]
```

and further below

```
93c93
< set pmesh_top_str_pitch [expr 10 * 10 * $M1_route_pitchX]
---
> set pmesh_top_str_pitch [expr 4 * 10 * $M1_route_pitchX]
```

Can you check if freepdk-45nm is fine with 4 instead of 10? If you bring up the layout with 4, and it looks perfectly fine, we should just use *4 as the arbitrary pitch.